docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+      </sect3>
+
+      <sect3 id="YML_TheDataCategory">
+        <title>Cluster Disambiguation</title>
+
+        <para>In the past we referred to data and storage as a cluster. We
+        overused the term cluster, and created some confusion around that
+        term. In the container space, it doesn't make any sense to say cluster
+        when really talking about a data plane. For data storage with nodes,
+        such as Thor running on those nodes, we will use the term, plane, as
+        in data plane, or storage plane. Aligning more so with industry
+        standards in the container space. When specifying where a workunit
+        would run we will use the tern queue. Doing so differentiates two
+        former uses of the term cluster, which will help in reducing some of
+        that confusion.</para>
+      </sect3>
+    </sect2>
+
+    <sect2>
+      <title>The Security Values</title>
+
+      <para>This section will look at the values.yaml sections dealing with
+      the system security components.</para>
+
+      <sect3>
+        <title>Certificates</title>
+
+        <para>The certificates section can be used to enable the cert-manager
+        to generate TLS certificates for each component in the HPCC Systems
+        deployment.</para>
+
+        <programlisting>certificates:
+  enabled: false
+  issuers:
+    local:
+      name: hpcc-local-issuer</programlisting>
+
+        <para>In the delivered yaml file certificates are not enabled, as
+        illustrated above. You must first install the cert-manager to use this
+        feature.</para>
+      </sect3>
+
+      <sect3 id="ValYAML_Secrets">
+        <title>Secrets</title>
+
+        <para>The Secrets section contains a set of categories, each of which
+        contain a list of secrets. The Secrets section is where to get info
+        into the system if you don't want it in the source. Such as code with
+        embedded code, you can have that defined in the code sign sections. If
+        you have information that you don't want public but need to run it you
+        could use secrets.</para>
+      </sect3>
+
+      <sect3>
+        <title>Vaults</title>
+
+        <para>Vaults is another way to do Secrets. The vaults section mirrors
+        the secret section but leverages <emphasis>HashiCorp Vault</emphasis>
+        for the storage of secrets. There is an additional category for vaults
+        named "ecl-user". The intent of the ecl-user vault secrets is to be
+        readable directly from ECL code. Other secret categories are read
+        internally by system components and not exposed directly to ECL
+        code.</para>
+      </sect3>
+    </sect2>
+  </sect1>
+
+  <sect1 id="TheValuesYaml_FileAndRelated" role="nobrk">
+    <title>The HPCC Components in the values.yaml File</title>
+
+    <para>The HPCC Systems Helm charts all ship with stock/default values.
+    These delivered stock Helm chart has a set of default values ideally to be
```

Comment:
> These delivered stock Helm chart has a set of default values ideally to be used as a guide in configuring your deployment.

'delivered stock helm chart' .. is confusing.

It's just a standard helm chart, with defaults (as all helm charts have).